components/crud-web-apps/jupyter/frontend/package.json

```diff
@@ -32,10 +32,10 @@
     "@fortawesome/free-brands-svg-icons": "^5.12.0",
     "@fortawesome/free-solid-svg-icons": "^5.12.0",
     "@kubernetes/client-node": "^0.12.2",
-    "hammerjs": "^2.0.8",
     "date-fns": "^1.29.0",
-    "lodash-es": "^4.17.11",
```

Comment:
Let's exclude this change from this PR and keep the touched lines to the minimum. This will also remove the 15.000 modified lines to the `package-lock.json` because of this change.

Although I'd like to progressively replace the use of `lodash` with `lodash-es`, but until I get to it we can remove this from the `package.json`, but lets just do it in another PR.